BUG in Build

Build Error: go test -c -o /Users/enzii/Projects/agents-lab/pkg/deepseek/__debug_bin104204876 -gcflags all=-N -l .
# github.com/yichozy/deepseek/request
../../../../go/pkg/mod/github.com/yichozy/deepseek@v1.0.1/request/validator.go:117:17: invalid operation: req.TopP != nil (mismatched types float32 and untyped nil)
../../../../go/pkg/mod/github.com/yichozy/deepseek@v1.0.1/request/validator.go:118:9: invalid operation: cannot indirect req.TopP (variable of type float32) (exit status 1)

--- a/request/request_test/validator_test.go
+++ b/request/request_test/validator_test.go
@@ -30,7 +30,7 @@ func TestValidateChatCompletionsRequest(t *testing.T) {
 			IncludeUsage: true,
 		},
 		Temperature: 2.0,
-		TopP:        nil, // TODO: VN -- pass non nil
+		TopP:        float32Ptr(0.5), // Pfa8a
 	}
 	err := request.ValidateChatCompletionsRequest(req)
 	assert.NoError(t, err)

